### PR TITLE
Remove references to simple-xmpp

### PR DESCRIPTION
**Pull request by silverbucket**
Updates API.md for new function signatures and removes references to simple-xmpp

### CODE REVIEW

File: packages/sockethub-platform-xmpp/API.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Connect to the XMPP server.
 ```
 <a name="XMPP+join"></a>
 
-### xmpP.join(job, credentials, done)
+### xmpP.join(job, done)
```
Comment (galfert) [inline packages/sockethub-platform-xmpp/API.md]:
```suggestion
### xmpp.join(job, done)
```

File: packages/sockethub-platform-xmpp/API.md [hunk 2]
```diff
@@ -142,7 +141,7 @@ Join a room, optionally defining a display name for that room.
 ```
 <a name="XMPP+send"></a>
 
-### xmpP.send(job, credentials, done)
+### xmpP.send(job, done)
```
Comment (galfert) [inline packages/sockethub-platform-xmpp/API.md]:
```suggestion
### xmpp.send(job, done)
```

File: packages/sockethub-platform-xmpp/API.md [hunk 3]
```diff
@@ -198,15 +196,14 @@ Send a message to a room or private conversation.
 ```
 <a name="XMPP+update"></a>
 
-### xmpP.update(job, credentials, done)
+### xmpP.update(job, done)
```
Comment (galfert) [inline packages/sockethub-platform-xmpp/API.md]:
```suggestion
### xmpp.update(job, done)
```

File: packages/sockethub-platform-xmpp/API.md [hunk 4]
```diff
@@ -226,15 +223,14 @@ Indicate presence and status message.
 ```
 <a name="XMPP+request-friend"></a>
 
-### xmpP.request-friend(job, credentials, done)
+### xmpP.request-friend(job, done)
```
Comment (galfert) [inline packages/sockethub-platform-xmpp/API.md]:
```suggestion
### xmpp.request-friend(job, done)
```

File: packages/sockethub-platform-xmpp/API.md [hunk 5]
```diff
@@ -252,15 +248,14 @@ Send friend request
 ```
 <a name="XMPP+remove-friend"></a>
 
-### xmpP.remove-friend(job, credentials, done)
+### xmpP.remove-friend(job, done)
```
Comment (galfert) [inline packages/sockethub-platform-xmpp/API.md]:
```suggestion
### xmpp.remove-friend(job, done)
```

File: packages/sockethub-platform-xmpp/API.md [hunk 6]
```diff
@@ -278,15 +273,14 @@ Send a remove friend request
 ```
 <a name="XMPP+make-friend"></a>
 
-### xmpP.make-friend(job, credentials, done)
+### xmpP.make-friend(job, done)
```
Comment (galfert) [inline packages/sockethub-platform-xmpp/API.md]:
```suggestion
### xmpp.make-friend(job, done)
```

File: packages/sockethub-platform-xmpp/API.md [hunk 7]
```diff
@@ -304,7 +298,7 @@ Confirm a friend request
 ```
 <a name="XMPP+observe"></a>
 
-### xmpP.observe(job, credentials, done)
+### xmpP.observe(job, done)
```
Comment (galfert) [inline packages/sockethub-platform-xmpp/API.md]:
```suggestion
### xmpp.observe(job, done)
```

**Review by galfert — CHANGES_REQUESTED** [review]
Left a couple comments.

File: packages/sockethub-platform-xmpp/src/index.js [hunk 8]
```diff
@@ -460,8 +460,8 @@ class XMPP {
   };
 
   cleanup(done) {
-    // FIXME - review this, simple-xmpp has a close func now i believe
-    this.debug('should be CLOSING connection now, NOT IMPLEMENTED in node-xmpp');
+    // FIXME - review this, there should be a close function of some sort
```
Comment (galfert) [inline packages/sockethub-platform-xmpp/src/index.js]: I think the function you are looking for is `stop()`. See https://github.com/xmppjs/xmpp.js/tree/312838782c0b127ffabaa19ff58ad3aaf71a295d/packages/client#stop.

Comment (silverbucket) [reply packages/sockethub-platform-xmpp/src/index.js]: thanks, just pushed the change

**Comment by silverbucket** [comment]
@galfert the API doc generation capitalizes that xmpP for some reason, it's annoying. Anyway, I've updated based on your comments. Should be ready for review again after the tests pass.

**Review by galfert — APPROVED** [review]
👍

**Pull request merged**